main.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+import discord
+
+TOKEN = os.getenv('DISCORD_TOKEN')
+GUILD = os.getenv('DISCORD_GUILD')
+
+client = discord.Client()
+
+@client.event
+async def on_ready():   
+    print(str(client.user) + ' has connected to Discord successfully.')
```

Comment:
It's more common in python3.x to interpolate variables by way of f-strings to improve readability and reduce the number of explicit programming instructions.

Here you are explicitly using the `str()` and the `+` operators. These instructions are implicit to f-strings as the following suggestion demonstrates:

```suggestion
    print(f'{client.user} has connected to Discord successfully.')
```